security/pkg/server/ca/authenticate/kubeauth/kube_jwt.go

```diff
@@ -145,6 +153,10 @@ func (a *KubeJWTAuthenticator) getKubeClient(clusterID cluster.ID) kubernetes.In
 		return a.kubeClient
 	}
 
+	if !features.RemoteClusterAccess {
+		return a.kubeClient
```

Comment:
I donot think auth jwt with the config cluster work. The token proxy uses come from the cluster where it reside

Reply:
Right. It will not be authenticated if the JWT comes from a remote cluster - as intended ( remote cluster access disabled ). 

However if the token is from the current cluster - but has some missconfigured header - it will work, that's what I'm trying to do - the cluster header and clustername can be out of sync ( for example changing cluster name from Kubernetes default to a real one ). This reduces the risks.